Target chamber, target exit window and connecting pipe implementation

sayak-alt opened the pull request:
The target chamber, target exit window (https://moller.jlab.org/DocDB/0010/001007/001/TGT-1061-0070-0006.pdf) and connecting pipe from the target exit window to bellow
![target_chamber_1](https://user-images.githubusercontent.com/67478639/234438121-001a3719-b0a4-4c98-b029-599936c14858.jpg)
1 are implemented. Here are a couple of screenshots from the engineering drawings and JT file.
![target_chamber_exit_window](https://user-images.githubusercontent.com/
![target_chamber_2](https://user-images.githubusercontent.com/67478639/234438098-fc80bfa7-1ccb-4f31-b7b7-851a039edf20.jpg)
![connecting_pipe](https://user-images.githubusercontent.com/67478639/234438111-b8c17517-08a9-4da5-bece-de69c897b2a0.jpg)
67478639/234437949-8aa3a880-6e3e-45a3-b0c8-a6bc6e40789f.png)


GDML screenshots;


![target_chamber_implementation_gdml_fig5](https://user-images.githubusercontent.com/67478639/234688302-5d7600f2-d62b-4023-8e8e-22b6503ee07a.png)
![target_chamber_implementation_gdml_fig4](https://user-images.githubusercontent.com/67478639/234688306-166775c8-6321-4057-b2d0-3d2c871574ae.png)
![target_chamber_implementation_gdml_fig3](https://user-images.githubusercontent.com/67478639/234688308-19197cbd-4c20-4813-a8d7-62fc4249ff47.png)
![target_chamber_implementation_gdml_fig2](https://user-images.githubusercontent.com/67478639/234688311-900ab58c-d273-4488-8d03-50538e5eccf7.png)
![target_chamber_implementation_gdml_fig1](https://user-images.githubusercontent.com/67478639/234688316-cfe75b2a-0ca8-495a-9ecb-32f0ff768613.png)


Implemented the hole (similar to the JT file) for the beam entrance on the target chamber boundary. Please see the attached images for reference:

JT file images:

![beam_entrance_hole_target_chamber_jt_image](https://user-images.githubusercontent.com/67478639/236065293-519dcf58-aab7-4412-8a6b-bebc93dce2d8.png)






GDML images:
![beam_entrance_hole_target_chamber_gdml_image_1](https://user-images.githubusercontent.com/67478639/236065325-50c3065c-44d2-42b1-9bc7-b8370e7432eb.png)
![beam_entrance_hole_target_chamber_gdml_image_2](https://user-images.githubusercontent.com/67478639/236065326-12584981-9703-4520-9a8c-42ca93b9b79c.png)
![beam_entrance_hole_target_chamber_gdml_image_3](https://user-images.githubusercontent.com/67478639/236065327-e1ed780d-5d1a-4f73-8323-b6bda9f24991.png)

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update hybridDaughter.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Update subTargetRegion.gdml

sayak-alt committed:
Target chamber, target exit window & connection pipe

The target chamber, target exit window (https://moller.jlab.org/DocDB/0010/001007/001/TGT-1061-0070-0006.pdf) and the pipe connecting the target exit window and the bellow 1 are implemented.

rahmans1 commented on geometry/hybrid/hybridDaughter.gdml:
@sayak-alt Can you revert this change.  Hybrid daughter is not relevant to this PR.

sayak-alt committed:
Update hybridDaughter.gdml

rahmans1 commented:
> The target chamber, target exit window (https://moller.jlab.org/DocDB/0010/001007/001/TGT-1061-0070-0006.pdf) and connecting pipe from the target exit window to bellow ![target_chamber_1](https://user-images.githubusercontent.com/67478639/234438121-001a3719-b0a4-4c98-b029-599936c14858.jpg) 

@sayak-alt There is a hole in the center on the floor in JT whereas the GDML has a solid block. If it's not too hard it may be good to put it in. @cipriangal can comment whether it's ok to  leave it as it is in GDML.
![image](https://user-images.githubusercontent.com/7409132/234949619-58523fda-2f25-41e4-9070-5979eb5ec8d3.png)

@rahmans1 I thought since it is not in the beam direction it is not that important. If I am wrong then please let me know, I can implement the holes in the target chamber as it is in the JT file.

rahmans1 reviewed (approved):
Looks good mostly. Would have been better if the commit messages were a little more descriptive but in general gdml picture seems to reflect the drawing. Approving.

sayak-alt committed:
Update subTargetRegion.gdml

Implemented the entry window hole for the beam entrance at the target chamber boundary
